### ISSUE TITLE
No devices showing after install

### ISSUE THREAD

**Issue by grantwils** [ollo69/ha_tuya_custom#24]
Forgive me as I'm clearly doing something wrong. 

I installed this to get my colourwheel on my RGB spotlights. 
Firstly i uninstalled the standard tuya integration.
Then I installed ha_tuya_custom
This is my config
tuya_custom:
```
  username: <my@email>
  password: !secret tuya_pw
  devices_config:
    - device_name: US Spotlight - Murdo # this is mandatory with at least one of the other keys
      support_color: true # true or false, force color support for device
```

US Spotlight - Murdo <is the friendly name of an RGB spotlight which previously worked (without RGB)

I cannot get any of the previous devices to show with or without the device name. Nothing shows up anywhere in HA. 
I have no errors in logs.
What did I do wrong please?

**Comment by grantwils** [comment]
Discovered my error. ignore. sorry

**Comment by ollo69** [comment]
Share your error could help other similar situation

**Comment by grantwils** [comment]
It was one of three things or a combination of the three:
1. I was using my email address, not my mobile number.
2. The installation instructions aren't clear.
3. I'm a dumbass.

I decided to give up and removed ha_tuya_custom from my config file. I went to re-install Tuya integration from the integrations page and I noticed ha_tuya-custom was available to install (I'd just been adding to my config file like I did when i set up tuya). It wouldn't let me log in with email, so I entered my mobile number and then it all worked. 

Like I said, possibly a combination of all three. 👍 
Sorry to trouble you
